src/mapbox/geojson.cpp

```diff
@@ -75,22 +75,43 @@ geometry convert<geometry>(const json_value &json) {
 }
 
 template <>
+value convert<value>(const json_value &json);
+
+template <>
+std::unordered_map<std::string, value> convert(const json_value &json) {
+    if (!json.IsObject())
+        throw error("properties must be an object");
+
+    std::unordered_map<std::string, value> result;
+    for (auto itr = json.MemberBegin(); itr != json.MemberEnd(); ++itr) {
+        result[itr->name.GetString()] = convert<value>(itr->value);
```

Comment:
Should we use `emplace` here (and in one other place) to avoid default constructor on `result[]` operator, as @kkaefer suggested?